src/elements/hv-doc/index.ts

```diff
@@ -1,2 +1,2 @@
 export { default } from './hv-doc';
-export { Context as StateContext } from './context';
+export { Consumer as StateConsumer, useHvDoc } from './context';
```

Comment:
naming: what do you think if we were to match the hook and the consumer component (since they are equivalent for FC / Class components).
```suggestion
export { Consumer as HvDocContext, useHvDocContext } from './context';
```

Reply:
Yeah, that makes sense. I'll go over the others too and see if they are matching.